### DISCUSSION TITLE
[Enhancement]: Show Like and Dislike icon in song rows when viewing album

### DISCUSSION THREAD

**Discussion by squintgit** [ciderapp/Cider#794] [Feature Request]
### The feature you would like

When you Like or Dislike a song, you have to right-click the song to see if it's liked or disliked.

Would love to see Cider show the Like or Dislike icon on the actual song row so you can see this at a glance. (Or at least have an option to do so.)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cmpute** [comment]
I would like to further suggest that, the "liked" version of the like icon should be solid instead of hollow. Currently it's hard to distinguish liked and not liked songs

**Reply by KillerParticles** [reply]
Being able to see liked songs in the album row has been added in the latest build.

As for your comment, some themes like iTheme have the heart fully filled in instead of outlining it.

**Reply by cmpute** [reply]
Thanks for the reply! Unfortunately I prefer the default theme over the iTheme, but it's great to know that I can change it using the themes.
